src/alert/index.tsx

```diff
@@ -4,7 +4,7 @@ import React from 'react';
 import { applyDisplayName } from '../internal/utils/apply-display-name';
 import { AlertProps } from './interfaces';
 import InternalAlert from './internal';
-import useBaseComponent from '../internal/hooks/use-base-component';
+import { useBaseComponent } from '../internal/base-component';
```

Comment:
Any chances of not changing this in the current PR? The import path will change to `@cloudscape-design/component-toolkit` soon, so let's keep the current ones for now

Reply:
Yes, will update.